cli/src/main/java/com/devonfw/tools/ide/commandlet/EditionSetCommandlet.java

```diff
@@ -35,9 +31,7 @@ public EditionSetCommandlet(IdeContext context) {
     addKeyword(getName());
     this.tool = add(new ToolProperty("", true, "tool"));
     this.edition = add(new EditionProperty("", true, "edition"));
-    this.conf = add(new FlagProperty("--conf", false, null));
-    this.home = add(new FlagProperty("--home", false, null));
-    this.workspace = add(new FlagProperty("--workspace", false, null));
+    this.cfg = add(new EnumProperty<>("cfg", false, null, EnvironmentVariablesType.class));
```

Comment:
IMHO you need to make this an option as otherwise it will later be tricky to maintain the CLI if other args may be added.
Now, I could provide `conf` or `user` as last argument to make it optional. But only if we go for `--cfg=conf` or `--cfg=user` we will have the ability also in the future to have multiple such optional CLI aspects since assuming we would later add an additional optional `StringProperty` how could our CLI parser know if `conf` should be the value for `cfg` or for the new `StringProperty` argument?

```suggestion
    this.cfg = add(new EnumProperty<>("--cfg", false, null, EnvironmentVariablesType.class));
```